style.css

```diff
@@ -1,5 +1,1007 @@
+    width: 99vmax;
+    height: 846px;
+    align-items: center;
+    margin: 0 auto;
+    text-align: center;
+    justify-content: center;
+  }
+
+  .first-title-3 {
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: bold;
+    font-size: 40px;
+    line-height: 52px;
+
+    /* or 130% */
+    height: 56px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    margin: 0;
+
+    /* Neutrals/N0 */
+
+    color: #fff;
+  }
+
+  .description-4 {
+    font-family: "poppins", sans-serif;
+    width: 569px;
+    height: 102px;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #ebebff;
+  }
+
+  .contact-form {
+    display: flex;
+    flex-direction: column;
+  }
+
+  input {
+    background: #fff;
+    width: 447px;
+    height: 48px;
+    margin: 10px 0 10px 0;
+    padding-left: 20px;
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 17px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    color: #172b4d;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+  }
+
+  textarea {
+    background: #fff;
+    width: 447px;
+    height: 170px;
+    padding-left: 20px;
+    padding-top: 10px;
+    margin-top: 10px;
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 24px;
+    color: #b3bac5;
+    border: 1px solid #cfd8dc;
+    border-radius: 2px;
+  }
+
+  .highlight-4 {
+    background: #fff;
+    border-radius: 8px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    border: 1px solid white;
+    width: 150px;
+    margin-top: 15px;
+    margin-left: auto;
+    margin-right: auto;
+  }
+
+  .font-6 {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: #6070ff;
+  }
+
+  .highlight-4:hover {
+    background: #6070ff;
+    border-radius: 8px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    border: 1px solid #6070ff;
+    width: 150px;
+    margin-top: 15px;
+  }
+
+  .font-6:hover {
+    font-family: "poppins", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 24px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    letter-spacing: 0.03em;
+    color: white;
+  }
+
+  .highlight-4:visited {
+    background: #2230d2;
+    border-radius: 8px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 12px 16px;
+    border: 1px solid #2230d2;
+    width: 150px;
+    margin-top: 15px;
+  }
```

Comment:
- [ ] Great job implementing the button interactions but after I clicked on one of your buttons, all the buttons on your page indicated that I had clicked on them. It'll be nice to address this issue so as to have a better user interface 💯

Reply:
Addressed ✔